components/Frame/Header.js

```diff
@@ -179,14 +189,31 @@ const isPositionStickySupported = () => {
   return style.position.indexOf('sticky') !== -1
 }
 
+// Workaround for WKWebView fixed 0 rendering hickup
+// - iOS 11.4
+const forceRefRedraw = ref => {
+  if (ref) {
+    setTimeout(() => {
+      const display = ref.style.display
+      ref.style.display = 'none'
+      /* eslint-disable-next-line no-unused-expressions */
+      ref.offsetHeight
+      ref.style.display = display
+    }, 300)
+  }
+}
+
 class Header extends Component {
   constructor (props) {
     super(props)
 
     this.state = {
       opaque: !this.props.cover,
       mobile: false,
-      expanded: false
+      expanded: false,
+      hasBackButton: props.inNativeIOSApp
+        ? undefined
```

Comment:
Optional:
The fine distinction between false and undefined may be a source of errors in the code below in the future, but well, ...